core/src/avm2/array.rs

```diff
@@ -114,6 +114,25 @@ impl<'gc> ArrayStorage<'gc> {
         }
     }
 
+    /// Replace the existing dense storage with a new dense storage
+    /// Does nothing if this `ArrayStorage` is sparse
```

Comment:
This comment is now outdated due to the added `panic!`.
I'm just gonna fix this and merge.
Thank you!

Reply:
... also squashed this last "chore" commit into the first one.